src/components/page/StandardPage.tsx

```diff
@@ -159,7 +162,7 @@ export type StandardPageProps<T> = {
    * Extract value from fields and compare to selected filter.
    * The default matchers support the default filters.
    */
-  extraSupportedMatchers?: ValueMatcher[];
+  extraSupportedMatchers?: ValueMatcher<any>[];
```

Comment:
Cant you get rid of this any?

Reply:
If not let me know and we will merge the PR

Reply:
Updated.